nautobot/dcim/migrations/0001_initial_part_1.py

```diff
@@ -526,10 +526,10 @@ class Migration(migrations.Migration):
                 ("serial", models.CharField(blank=True, max_length=50)),
                 ("asset_tag", models.CharField(blank=True, max_length=50, null=True, unique=True)),
                 ("discovered", models.BooleanField(default=False)),
-                ("lft", models.PositiveIntegerField(editable=False)),
-                ("rght", models.PositiveIntegerField(editable=False)),
-                ("tree_id", models.PositiveIntegerField(db_index=True, editable=False)),
-                ("level", models.PositiveIntegerField(editable=False)),
+                ("lft", models.PositiveIntegerField(default=0, editable=False)),
+                ("rght", models.PositiveIntegerField(default=0, editable=False)),
+                ("tree_id", models.PositiveIntegerField(default=0, editable=False)),
+                ("level", models.PositiveIntegerField(default=0, editable=False)),
```

Comment:
Addition of defaults here is to make reverse-migration (rolling back the dcim/0021 migration) possible - since MPTT is no longer auto-populating these fields, they needed to have a defined default value else the DB complains about trying to set them to null when doing the reverse migration.

Reply:
While I think it's extremely likely that if you upgrade to 2.0 you won't really be able to roll back, it's good to try.